mlflow/gateway/schemas/chat.py

```diff
@@ -76,13 +87,33 @@ class ChatUsage(ResponseModel):
 }
 
 
-class ResponsePayload(ResponseModel):
-    id: Optional[str] = None
-    object: str = "chat.completion"
-    created: int
-    model: str
+class Function(_Function, ResponseModel):
```

Comment:
note: putting these classes back to ensure every Gateway response models inherits `ResponseModel`. It is equivalent with `BaseModel` functionality wise and should not be used by users, but for consistency.

ref: https://github.com/mlflow/mlflow/blob/v2.19.0/mlflow/gateway/schemas/chat.py